backend/pom.xml

```diff
@@ -16,7 +16,7 @@
         <maven.compiler.target>17</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <frontend-bundle-dir>../frontend/dist/project-template/</frontend-bundle-dir>
-        <start-class>de.davidemarcoli.backend.BackendApplication</start-class>
+        <start-class>com.social.dialylink.BackendApplication</start-class>
```

Comment:
com.social.dialylink to com.social.dailylink

Reply:
Fixed in following commit